bundles/org.eclipse.orion.client.ui/web/plugins/iframeCommunicationPlugin.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Yeah, their doc says so:
https://wiki.eclipse.org/Orion/Documentation/Developer_Guide/Simple_plugin_example

Also, I inlined the service impl code so we don't need to register it.